CHANGELOG.md

```diff
@@ -8,43 +8,82 @@ A breaking change will get clearly marked in this log.
-  - the `asset_type` field can now be `liquidity_pool_shares`.
-  - `buying_liabilities`, `selling_liabilities`, `asset_code`, and `asset_issuer` are omitted from the response for pool shares because they are not relevant to liquidity pools.
-- Update the `ChangeTrustOperationResponse` interface so it can conform to a change in a liquidity pool trustline ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)).
-  - `asset_type` can now be `liquidity_pool_shares`.
-  - `asset_code` and `asset_issuer` are now optional.
-  - Added `liquidity_pool_id` as an optional field.
-- Trustline created/updated/revoked effects' asset type can now be `liquidity_pool_shares` ([#690](https://github.com/stellar/js-stellar-sdk/pull/690)).
-- Trustline sponsorship effects have been updated so the `asset` becomes optional and is replaced with `liquidity_pool_id` for liquidity pools ([#690](https://github.com/stellar/js-stellar-sdk/pull/690)).
-- The `trustee` attribute was made optional in the `ChangeTrustOperationResponse` ([#692](https://github.com/stellar/js-stellar-sdk/pull/692)).
+- A `TradeRecord` can now correspond to two different types of trades and has changed ([#685](https://github.com/stellar/js-stellar-sdk/pull/685)):
+  * `Orderbook` (the existing structure)
+    - `counter_offer_id` and `base_offer_id` only show up in these records
+  * `LiquidityPool` (new)
+    - `base_account` xor `counter_account` will appear in these records
+  * `price` fields changed from `number`s to `string`s
+  * The links to `base` and `counter` can now point to *either* an account or a liquidity pool
+
+- An account's `balances` array can now include a new type ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)):
+  * `asset_type` can now be `liquidity_pool_shares`
+  * The following fields are *not* included in pool share balances:
+    - `buying_liabilities`
+    - `selling_liabilities`
+    - `asset_code`
+    - `asset_issue`
+
+- The `ChangeTrustOperationResponse` has changed ([#688](https://github.com/stellar/js-stellar-sdk/pull/688)):
+  * `asset_type` can now be `liquidity_pool_shares`
+  * `asset_code` and `asset_issuer` are now optional
+  * `liquidity_pool_id` is a new optional field
+
+- The trustline effects (`TrustlineCreated`, `TrustlineUpdated`, `TrustlineRevoked`) have changed ([#690](https://github.com/stellar/js-stellar-sdk/pull/690)):
+  * the asset type can now be `liquidity_pool_shares` 
+  * they can optionally include a `liquidity_pool_id`
+
+- Trustline sponsorship effects (`TrustlineSponsorshipCreated`, `TrustlineSponsorshipUpdated`, `TrustlineSponsorshipRemoved`) have been updated ([#690](https://github.com/stellar/js-stellar-sdk/pull/690)): 
+  * the `asset` field is now optional, and is replaced by
+  * the `liquidity_pool_id` field for liquidity pools
+
+- The `ChangeTrustOperationResponse` has changed ([#692](https://github.com/stellar/js-stellar-sdk/pull/692)):
+  * the `trustee` field is now optional
```

Comment:
💡  There are two items referring to `ChangeTrustOperationResponse` breaking changes here. We should group them into a single item.

Reply:
Another excellent point :+1: done in 320ef433